pkg/collectors/builder.go

```diff
@@ -107,7 +102,8 @@ func (b *Builder) Build() []*metricsstore.MetricsStore {
 }
 
 var availableCollectors = map[string]func(f *Builder) *metricsstore.MetricsStore{
-	"managedclusterinfos": func(b *Builder) *metricsstore.MetricsStore { return b.buildManagedClusterInfoCollector() },
+	"managedclusterinfos":  func(b *Builder) *metricsstore.MetricsStore { return b.buildManagedClusterInfoCollector() },
+	"managedclusterlabels": func(b *Builder) *metricsstore.MetricsStore { return b.buildManagedClusterLabelCollector() },
```

Comment:
Also need to enable it as default in https://github.com/stolostron/clusterlifecycle-state-metrics/blob/main/pkg/options/collector.go#L21

Reply:
Okay, I have enabled this within an earlier commit. 👍🏾